src/apiCalls.js

```diff
@@ -0,0 +1,12 @@
+
+export const getEmailInfo = async (email) => {
+  // const response = await fetch(`https://apilayer.net/api/check?access_key=c9e97b26d4099cd4e226ce145b24ff95&email=${email}&format=1`)
+  
+  const response = await fetch(`https://www.thecocktaildb.com/api/json/v1/1/search.php?s=${email}`)
```

Comment:
testing on a free cocktail api so I don't go over the call limit for the actual email verify api